server/service/src/sync/translations/special/name_to_name_store_join.rs

```diff
@@ -50,12 +50,6 @@ impl SyncTranslation for NameToNameStoreJoinTranslation {
         let name_store_joins = NameStoreJoinRepository::new(connection)
             .query_by_filter(NameStoreJoinFilter::new().name_id(EqualFilter::equal_to(&data.ID)))?;
 
-        if name_store_joins.is_empty() {
```

Comment:
This was always empty... so couldn't get the names/name store joins anymore........